FusionIIIT/applications/central_mess/handlers.py

```diff
@@ -541,14 +541,14 @@ def add_mess_committee(request, roll_number):
     print("\n\n\\n\n\n\n\n")
     print(roll_number)
     mess = Messinfo.objects.get(student_id__id=roll_number)
-    print(mess)
+    print(mess) 
```

Comment:
Please remove unnecessary print statements.

Reply:
Removed In a new Commit